### PR TITLE
fix: Added translation for room visibility dropdown 

### PR DESCRIPTION
**Pull request by mananjadhav**
### Details
- Added translations for room visibility dropdown

### Fixed Issues

$ https://github.com/Expensify/App/issues/6733

### Tests
1. Verified translations in English as well as Spanish
2. Verified Room creation visibility values being passed.

### QA Steps
1. Go to Preferences -> change your language
2. On the home screen, click on the FAB and open the quick options menu
3. Click on New Room
4. Ensure that the visibility dropdown options are in the correct language translation

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

web-room-visibility-translation

#### Mobile Web

mweb-room-visibility-translation

#### Desktop

desktop-room-visibility-translation

#### iOS

ios-room-visibility-translation

#### Android

android-room-visibility-translation

### CODE REVIEW

File: src/pages/workspace/WorkspaceNewRoomPage.js [hunk 1]
```diff
@@ -129,6 +129,12 @@ class WorkspaceNewRoomPage extends React.Component {
             return null;
         }
         const shouldDisableSubmit = Boolean(!this.state.roomName || !this.state.policyID || this.state.error);
+
+        const visibilityOptions = _.map(_.values(CONST.REPORT.VISIBILITY), v => ({
```
Comment (parasharrajat) [inline src/pages/workspace/WorkspaceNewRoomPage.js:133]: Can you please use a more descriptive name?

**Review by parasharrajat — CHANGES_REQUESTED** [review]

**Comment by mananjadhav** [comment]
@parasharrajat  PR updated.

**Review by parasharrajat — APPROVED** [review]
LGTM

Cc:@puneetlath 


:ribbon: :eyes: :ribbon:

**Review by puneetlath — APPROVED** [review]
Looks good!

**Pull request merged**